assets/variables.css

```diff
@@ -7,7 +7,8 @@
 	--color-tertiary-text: #47133d;
 	--color-tertiary-border: #a17193;
 	--color-highlight: #f7cf56;
-	--color-highlight-alt: #a4cacb;
+	--color-highlight-alt-text: #a4cacb;
+	--color-highlight-alt-border: #ebf3f3;
```

Comment:
#43 suggests that this should be added to the editor palette. Is this PR supposed to take care of that too?

Reply:
Yes, as the `highlight-alt` color was added in #22. What this is doing is setting it so the one color is lighter for backgrounds while text remains the same. I can't find a way to declare this via Gutenberg.

Reply:
I see, yeah, the palette supplied by the theme is used for both the background and text color. Why not just add the color explicitly to the palette? 

How it's done in this PR, the editor and front-end won't match:

**Editor**
<img width="1465" alt="Screen Shot 2020-05-14 at 12 06 47 AM" src="https://user-images.githubusercontent.com/5375500/81891609-dd516500-9576-11ea-9ce3-0e5ecf1da299.png">

**Front end**
<img width="830" alt="Screen Shot 2020-05-14 at 12 06 52 AM" src="https://user-images.githubusercontent.com/5375500/81891615-dfb3bf00-9576-11ea-9046-a16cced9b265.png">

Reply:
@jffng - Good point, may need to throw an `important` on there.

Regarding not creating a new color in the palette, as per the design, this is for backgrounds only so I don't want to clutter things up.

Reply:
@jffng - Tried to replicate but couldn't. Maybe a cache?

<img width="734" alt="Screen Shot 2020-05-14 at 10 00 57 PM" src="https://user-images.githubusercontent.com/1760168/82003461-6aa3c080-962e-11ea-8867-76b4983665be.png">

Reply:
Okay, in bb2ae3e I hacked Gutenberg to display the correct colors:
<img width="282" alt="Screen Shot 2020-05-15 at 10 49 26 PM" src="https://user-images.githubusercontent.com/1760168/82108727-8fffff80-96fe-11ea-8409-7e7603344958.png">
<img width="282" alt="Screen Shot 2020-05-15 at 10 49 32 PM" src="https://user-images.githubusercontent.com/1760168/82108728-90989600-96fe-11ea-9d65-4d7561b47cd4.png">

It's far from perfect but works for now imo.

Reply:
The picker is showing the correct color, but how the blocks actually display is still being overridden by the inline styles:

<img width="793" alt="Screen Shot 2020-05-18 at 9 34 22 AM" src="https://user-images.githubusercontent.com/5375500/82219078-cd28e500-98ea-11ea-96d4-ddcf6e19dbb1.png">

I think if you wanted to override the inline styles, there would need to be a rule like the following in the editor stylesheet:

```css
.has-hightlight-alt-background-color {
   background-color: var( --color-highlight-alt-border ) !important;
}
```

<img width="777" alt="Screen Shot 2020-05-18 at 9 37 41 AM" src="https://user-images.githubusercontent.com/5375500/82219446-4c1e1d80-98eb-11ea-90eb-48552aa57969.png">

Reply:
I was using an outdated version of Gutenberg, this LGTM as of 8.1.0.